tests/crafting_test.cpp

```diff
@@ -957,3 +924,35 @@ TEST_CASE( "book_proficiency_mitigation", "[crafting][proficiency]" )
         }
     }
 }
+
+TEST_CASE( "partial_proficiency_mitigation", "[crafting][proficiency]" )
+{
+    GIVEN( "a recipe with required proficiencies" ) {
+        clear_avatar();
+        clear_map();
+        const recipe &test_recipe = recipe_id( "leather_belt" ).obj();
```

Comment:
Exactly the same function, just a stylistic choice.
```suggestion
        const recipe &test_recipe = *recipe_id( "leather_belt" );
```

Reply:
Done.